### ISSUE TITLE
Add some useful examples of prerun modifiers to the docs

### ISSUE THREAD

**Issue by jaloren** [robotframework/robotframework#2299]
document how to use prerun modifiers to skip over setup/teardown. Also document how to skip over tests cases and test suites. Needs to include code examples.

**Comment by pekkaklarck** [comment]
I really liked the proposed enhancements. Some of the examples could be simplified if #2302 is implemented. Should we wait for that or should other enhancements be merged already now? Do you @jaloren have time to work with this in the near future? I have time this and next week but then I'll be one week away.

**Comment by jaloren** [comment]
I have time to work on this doc enhancement. I'd vote for waiting on the fix for 2302 and then simplifying the examples accordingly.

**Comment by pekkaklarck** [comment]
Cool, let's wait for #2302 and related #2303. They both ought to be pretty simple, but I doubt I have time for them this week and next week I'll be on holiday without much Internet access. I can implement them when I'm back, but you or someone else can obviously create PRs for them already before.

**Comment by pekkaklarck** [comment]
Do you @jaloren have time to look at this in the near future? Both #2302 and #2303 are done.

**Comment by jaloren** [comment]
@pekkaklarck  sure. I think i could submit a PR in the next two weeks for this. Would that work?

**Comment by pekkaklarck** [comment]
Sounds good. It's unlikely RF 3.0.1 is release in two weeks so we ought to get enhancements in, but this being postponed to 3.0.2 wouldn't be a disaster either. Notice that you can also update the existing PR #2298 by just adding new commits to that branch.

**Comment by jaloren** [comment]
@pekkaklarck the PR is done. See #2416. I did submit it on the same prerun modifier branch but it popped up different PR. Anyway, I think this will address the issue opened here and includes the changes that were made so it shows setup and teardown as directly settable.

**Comment by pekkaklarck** [comment]
Holidays, house moves, etc. has got on the way, but I'm now finally trying to get RF 3.0.1 ready. What's the status of this issue? PR  #2416 seems to be closed.

**Comment by jaloren** [comment]
woops ....it shouldn't be. Its ready to be merged though. So i'll reopen it.

**Comment by jaloren** [comment]
@pekkaklarck i believe PR #2473 can be merged into master. Please let me know if there are any issues.

**Comment by pekkaklarck** [comment]
Have been really busy the whole autumn and now try to get 3.0.1 out still this year. On a quick look PR  #2473 looks pretty OK but has some problems in examples I'd like to see fixed before merging. Better to postpone this to 3.0.2 than trying to rush it in.

**Comment by pekkaklarck** [comment]
New examples and related docs were provided by PR #2473 and pre-run modifier docs in general cleaned up and enhanced by 41b08341c67e35539cabe16458061db1691340fd.

Could you @jaloren take a look at how the examples and related docs look like after my changes?
